.github/mergify.yml

```diff
@@ -4,7 +4,7 @@ pull_request_rules:
       - author~=^dependabot\[bot\]$
       - check-success~=lint
       - check-success~=test
-      - title~=bump [^\s]+ from ([\d]+)\..+ to \1\.
+      - dependabot-update-type = version-update:semver-minor
```

Comment:
this one does from minor all the way to major right? if not please dismiss my approval!

Reply:
no, only `semver-minor`

Reply:
> Misunderstood this pr, what about patches ya?

i think lets just do 
```ts
- -dependabot-update-type = version-update:semver-major
```

which auto-merges everything less `semver-major`
![Screenshot 2024-04-02 at 7 32 23 PM](https://github.com/isomerpages/isomercms-backend/assets/44049504/e309c5e9-36ad-436e-92d3-ccd8ca909b76)